src/main/java/frc/robot/Robot.java

```diff
@@ -48,6 +52,15 @@
   boolean photoswitchSensorToggle = true;
   boolean shooterToggle = true;
   boolean drivetrainToggle = true;
+  boolean navXToggle = true;
+
+
+  double pitch;
```

Comment:
I'd advise just leaving these out until we need them to actually deal with the robot. It'll probably be cleaner to avoid keeping these in temp variables unless we need the same value in multiple places, and it's likely that the gyro itself avoids re-calculating the values unnecessarily between multiple calls.